projects/Mallard/src/hooks/use-issue.ts

```diff
@@ -41,39 +32,18 @@ export const useIssueResponse = (issue: {
         [issue],
     )
 
-export const getFrontsResponse = (
-    localIssueId: Issue['localId'],
-    publishedIssueId: Issue['publishedId'],
-    front: Front['key'],
-) =>
-    fetchFromIssue<Front>(
-        localIssueId,
-        FSPaths.front(localIssueId, front),
-        APIPaths.front(publishedIssueId, front),
-        {
-            validator: res => res.collections != null,
-        },
-    )
-
-export const useFrontsResponse = (
-    localIssueId: Issue['localId'],
-    publishedIssueId: Issue['publishedId'],
-    front: Front['key'],
-) =>
-    useIssueWithResponse(
-        getFrontsResponse(localIssueId, publishedIssueId, front),
-        [localIssueId, publishedIssueId, front],
-    )
-
 export const getArticleResponse = ({
     article,
     localIssueId,
     publishedIssueId,
     front,
 }: PathToArticle) =>
-    chain(getFrontsResponse(localIssueId, publishedIssueId, front), front => {
+    chain(getIssueResponse(localIssueId, publishedIssueId), issue => {
+        const maybeFront = issue.fronts.find(f => f.key === front)
+        if (!maybeFront) throw ERR_404_REMOTE
```

Comment:
`ERR_404_REMOTE` is a string, and I'd strongly discourage to `throw` strings; most notably because it does not provide the `stack`, so it wouldn't go into Sentry correctly afaik.

Reply:
This is a good point, it was like that before but can change.